package.json

```diff
@@ -39,6 +39,13 @@
     "envs": [
       "node",
       "browser"
-    ]
+    ],
+    "rules": {
+      "new-cap": ["error", {
+          "newIsCap": true,
+          "capIsNew": true,
+          "properties": false
+      }]
```

Comment:
This is removed now, but I've added it because eslint doesn't allow the uppercase function names for the default actions without the new keyword.